docs/_layouts/tutorial_frame.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+	<base target="_top">
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
off-topic: `<meta charset>` should be present in the first 1024 bytes of the document: https://web.dev/charset/